src/commonRoutes.tsx

```diff
@@ -29,6 +25,10 @@ export const commonRoutes: Route[] = [
   {
     path: 'account/:address/*',
     element: <AccountPage />,
+    errorElement: <ErrorBoundary></ErrorBoundary>,
+    loader: async ({ params }) => {
```

Comment:
async not needed anymore

Reply:
Oddly it works that way too, but all documentation and router code looks like it intends loader to return a Response or Promise. I'd keep it